src/Bicep.LangServer/Snippets/Templates/res-logic-app-connector.bicep

```diff
@@ -0,0 +1,9 @@
+﻿// Logic App Connector
+resource logicAppConnector 'Microsoft.Web/connections@2015-08-01-preview' = {
+  name: '${1:logicAppConnector}'
+  location: resourceGroup().location
+  properties: {
+    name: '${1:logicAppConnector}'

```

Comment:
Generally, I'm wondering if we should include the single-quotes in the snippet text. Some customers may be wanting to use variables or functions rather than string literals.

Reply:
I don't see any issues in not having the single-quotes. @majastrz , do you have any concerns?

Reply:
Makes sense to me.

Reply:
Removed the quotes here:
https://github.com/Azure/bicep/commit/6df25703d2225b86f8ff28bfa51e8fdb07d79ef0